plugins/covid/lab.py

```diff
@@ -145,3 +213,48 @@ def positive(test):
     covid_test = _get_covid_test(test)
     result = get_result(test)
     return result in covid_test.POSITIVE_RESULTS
+
+
+def get_covid_result_ticker(patient):
+    """
+    Given a PATIENT, return a list of dictionaries representing the last
+    3 results for each test type, with a timestamp, result string, and test name.
+    """
+    ticker = []
+
+    for test in COVID_19_TESTS:
+        data = []
+
+        tests = lab_test_models.LabTest.objects.filter(
+            test_name=test.TEST_NAME, patient=patient
+        ).order_by('-datetime_ordered')
+
+        for test in tests:
+            if len(data) > 2:
+                continue
+
+            if not resulted(test):
+                continue
+
+            timestamp = get_resulted_datetime(test).strftime('%d/%m/%Y %H:%M')
+            value     = get_result(test)
+            specimen  = get_specimen_type(test)
+
+            result_string = "{}".format(value)
+
+            if specimen:
+                result_string += " ({})".format(specimen)
+
+            data.append(
+                {
+                    'timestamp': timestamp,
+                    'name': _get_covid_test(test).OBSERVATION_NAME,
+                    'value': result_string
+                }
+            )
+
+        ticker += data
+
+    ticker = sorted(ticker, key=lambda i: i['timestamp'])
```

Comment:
This looks wrong, if we have 30/05/2020, 31/05/2020, 01/06/2020 it will reorder them 01/06/2020, 30/05/2020, 31/05/2020no?